app/models/concerns/course/assessment/submission/notification_concern.rb

```diff
@@ -17,6 +17,10 @@ def send_attempt_notification
 
   def send_submit_notification
     return unless workflow_state_before_last_save == 'attempting'
+    # When changing attempting to submitted state by a course staff,
+    # the creator (student) will be different with the updater (course staff).
+    # As such, there is no need to send a submit notification.
+    return unless creator == updater
```

Comment:
I think the comment here is not clear enough, and I was a bit confused as to what it means. Can be much more explicit.

Also, what about the case where I actually go to a student's attempt and submit it for them? I think the student should get notified in this case, else it will be quite shocking for them to find out later that someone has submitted for them.

Reply:
Noted - will update the comment.

The submit notification here is actually for the instructor. Normally, when a student submits a submission, instructors will be notified for the submission to be graded. Over here, since it's force submitted by the instructor himself, theres no need to resend another email for them to grade it

As of now even when an individual submission is submitted by an instructor, there is no notification sent to the student. I think we can add it for both force submit all and individual finalize submission case.